### PR TITLE
Update Linux.md as per the guidelines

### PR DESCRIPTION
**Pull request by pankaj892**

### CODE REVIEW

File: Linux.md [hunk 1]
```diff
@@ -96,3 +101,18 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="clearTerminalScreen"></a> Clears the screen of terminal
```
Comment (ghost) [inline Linux.md]: very common, pls remove this command.

File: Linux.md [hunk 2]
```diff
@@ -96,3 +101,18 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="clearTerminalScreen"></a> Clears the screen of terminal 
+`ctrl+L`
+
+### <a name="lastSudo"></a> Runs Previous Sudo command
+`sudo!!`
```
Comment (ghost) [inline Linux.md]: just put `$!!`

File: Linux.md [hunk 3]
```diff
@@ -96,3 +101,18 @@
...
+
+### <a name="randomSelect"></a> Randomly select files or lines from a folder
+`shuf`
```
Comment (ghost) [inline Linux.md]: Give a use-case where we can use it.

File: Linux.md [hunk 4]
```diff
@@ -96,3 +101,18 @@
...
+
+### <a name="showStat"></a> Show status and information of a file
+`stat`
```
Comment (ghost) [inline Linux.md]: Kindly remove the command. I am not finding a context where one can use this.

**Review by ghost — CHANGES_REQUESTED** [review]

File: Linux.md (hunk 1 above)
Comment (pankaj892) [reply Linux.md]: Removed the command check the updated file @arverma007

File: Linux.md [hunk 5]
```diff
@@ -96,3 +101,21 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="lastSudo"></a> Runs Previous Sudo command
```
Comment (ghost) [inline Linux.md]: Change the heading name

File: Linux.md [hunk 6]
```diff
@@ -96,3 +101,21 @@
...
+`$!!`
+
+### <a name="burnIt"></a> ASCII art animation that generates a burning fire on the terminal
```
Comment (ghost) [inline Linux.md]: Do you really use it while working? hahaha

File: Linux.md [hunk 7]
```diff
@@ -96,3 +101,21 @@
...
+`aafire`
+
+### <a name="showSW"></a> Watch Star Wars in ASCII on terminal
```
Comment (ghost) [inline Linux.md]: pls remove these commands

File: Linux.md [hunk 8]
```diff
@@ -96,3 +101,21 @@
...
+`telnet towel.blinkenlights.nl`
+
+### <a name="dnsip"></a> DNS lookup / Find IP of a website
+`dig websitename`
+
+### <a name="randomFake"></a> Generate random and possible fake identites
```
Comment (ghost) [inline Linux.md]: remove this aswell

File: Linux.md [hunk 9]
```diff
@@ -96,3 +101,21 @@
...
+`rig`
+
+### <a name="showCpu"></a> Show CPU information and architecture
```
Comment (ghost) [inline Linux.md]: Even this

**Review by ghost — CHANGES_REQUESTED** [review]
* The commands should not be copy-pasted from any source in bulk.
* Only add those commands that you use more frequently but may be unknown to other developers.

Example: `pwd`, `ls` e.t.c., are not allowed
* Follow the structure and don't forget to embed any reference links either in heading or command description.
    * Put it inside a directory if applicable
    * Give a proper heading
    * Use markdown script for [block code or inline code]((https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet#code)) to embed commands
* If the command heading is not sufficient to explain the uses, give 1 liner explanation with an example.
* I would be happy to accept your pull request even if you add `one` good command than adding `ten` not so good commands.

File: Linux.md (hunk 9 above)
Comment (pankaj892) [reply Linux.md]: I made the changes as per your suggestion review them @arverma007

File: Linux.md [hunk 10]
```diff
@@ -96,3 +100,18 @@
...
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSudoCommand"></a> Runs Previous Sudo command
+`$!!`
+
+### <a name="dnsip"></a> DNS lookup / Find IP of a website
+`dig websitename`
+
+### <a name="loggedInUser"></a>Give information of particular user
```
Comment (ghost) [inline Linux.md]: not required

File: Linux.md [hunk 11]
```diff
@@ -96,3 +100,18 @@
...
+`finger username`
+
+### <a name="memoryMap"></a> Display memory map of disk usage
```
Comment (ghost) [inline Linux.md]: not required

File: Linux.md [hunk 12]
```diff
@@ -96,3 +100,18 @@
...
+`pmap`
+
+### <a name="divideFileColumn"></a> Divide Files into Columns
```
Comment (ghost) [inline Linux.md]: not required

**Review by ghost — CHANGES_REQUESTED** [review]
2 of your commands are fine, remove the commented one and I will merge

File: Linux.md (hunk 12 above)
Comment (pankaj892) [reply Linux.md]: I have changed the file review it @arverma007

Comment (ghost) [reply Linux.md]: Remove this and change the heading for $!!

Comment (pankaj892) [reply Linux.md]: @arverma007 changed the heading check now

File: Linux.md [hunk 13]
```diff
@@ -96,3 +97,9 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSuperuserCommand"></a> Runs Previous command used as superuser
+`$!!`
+
+### <a name="divideFileColumn"></a> Divide Files into Columns
```
Comment (ghost) [inline Linux.md]: remove divideFileColumn command

**Review by ghost — CHANGES_REQUESTED** [review]

File: Linux.md (hunk 13 above)
Comment (pankaj892) [reply Linux.md]: Removed the command check now @arverma007

File: Linux.md [hunk 14]
```diff
@@ -96,3 +96,6 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSuperuserCommand"></a> Runs Previous command used as superuser
```
Comment (ghost) [inline Linux.md]: Improve heading to `Runs previous ran command`

**Review by ghost — CHANGES_REQUESTED** [review]

File: Linux.md (hunk 14 above)
Comment (pankaj892) [reply Linux.md]: Done @arverma007

**Review by ghost — APPROVED** [review]

**Pull request merged**